### PR TITLE
updated title and location of Tracking Protection to Firefox Tracking Protection

### LINKED ISSUE #19208: Privacy: Page title and location of document misleading.
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Privacy/Tracking_Protection

### What specific section or headline is this issue about?

Topic is browser specific

### What information was incorrect, unhelpful, or incomplete?

Title and location of document indicate that this page should be about privacy in general, but it is firefox specific. It should be in FF docs, not privacy docs.

### What did you expect to see?

It should be in FF docs, not privacy docs.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/privacy/tracking_protection`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Privacy/Tracking_Protection
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/privacy/tracking_protection/index.md
* Last commit: https://github.com/mdn/content/commit/d4b12e290fce9ae43a9ae23b9b9c8a5812b82ebd
* Document last modified: 2022-07-16T06:12:06.000Z

</details>

### PR DESCRIPTION
**Pull request by ghost**
#### Related issues

Fixes #19208 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
You added extra capitals to _Firefox_Tracking_Protection_. It must be _Firefox_tracking_protection_ (as I put in the command line in the related issue).

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -5760,7 +5760,7 @@
 /en-US/docs/Mozilla/Firefox/Privacy/Storage_Access_Policy/Errors/CookieBlockedForeign	/en-US/docs/Web/Privacy/Storage_Access_Policy/Errors/CookieBlockedForeign
 /en-US/docs/Mozilla/Firefox/Privacy/Storage_Access_Policy/Errors/CookieBlockedTracker	/en-US/docs/Web/Privacy/Storage_Access_Policy/Errors/CookieBlockedTracker
 /en-US/docs/Mozilla/Firefox/Privacy/Storage_Access_Policy/Errors/CookiePartitionedForeign	/en-US/docs/Web/Privacy/Storage_Access_Policy/Errors/CookiePartitionedForeign
-/en-US/docs/Mozilla/Firefox/Privacy/Tracking_Protection	/en-US/docs/Web/Privacy/Tracking_Protection
+/en-US/docs/Mozilla/Firefox/Privacy/Tracking_Protection	/en-US/docs/Web/Privacy/Firefox_Tracking_Protection
```
Comment (Josh-Cena) [inline files/en-us/_redirects.txt:5763]: You updated the slug, but not the redirect location

File: files/en-us/web/privacy/firefox_tracking_protection/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Tracking Protection
-slug: Web/Privacy/Tracking_Protection
+title: Firefox Tracking Protection
```
Comment (Josh-Cena) [inline files/en-us/web/privacy/firefox_tracking_protection/index.md:2]: This should be sentence case as well

Comment (teoli2003) [reply files/en-us/web/privacy/firefox_tracking_protection/index.md:2]: Yes, it should (and this one is easy to fix):
```suggestion
title: Firefox tracking protection
```

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (teoli2003) [reply files/en-us/_redirects.txt:5763]:
```suggestion
/en-US/docs/Mozilla/Firefox/Privacy/Tracking_Protection	/en-US/docs/Web/Privacy/Firefox_tracking_protection
```

File: files/en-us/_redirects.txt [hunk 3]
```diff
@@ -12274,6 +12274,7 @@
 /en-US/docs/Web/Performance/Time_to_interactive	/en-US/docs/Glossary/Time_to_interactive
 /en-US/docs/Web/Performance/business_case_for_performance	/en-US/docs/Learn/Performance/business_case_for_performance
 /en-US/docs/Web/Performance/learn	/en-US/docs/Web/Performance/How_browsers_work
+/en-US/docs/Web/Privacy/Tracking_Protection	/en-US/docs/Web/Privacy/Firefox_Tracking_Protection
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:12277]:
```suggestion
/en-US/docs/Web/Privacy/Tracking_Protection	/en-US/docs/Web/Privacy/Firefox_tracking_protection
```

**Comment by teoli2003** [comment]
Let's run the test to see if all is well.

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (teoli2003) [reply files/en-us/_redirects.txt:5763]:
```suggestion
/en-US/docs/Mozilla/Firefox/Privacy/Tracking_Protection	/en-US/docs/Web/Privacy/Firefox_tracking_protection
```

Comment (teoli2003) [reply files/en-us/_redirects.txt:5763]: Mmmh I can't commit this diff, the file is likely too large for GitHub. You'll need to make an extra push, @iSatVeerSingh

Comment (Josh-Cena) [reply files/en-us/_redirects.txt:5763]: @teoli2003 FYI only—if you really want to push to a fork via your local command line, you can always use the GH CLI (`gh pr checkout 19660`)

Comment (iSatVeerSingh) [reply files/en-us/_redirects.txt:5763]: I didn't changed _redirects.txt. Its automatically updated. I just run command

yarn content move Web/Privacy/Tracking_Protection Web/Privacy/Firefox_tracking_protection

**Comment by iSatVeerSingh** [comment]
What you guys have done, can you explain, please

**Comment by iSatVeerSingh** [comment]
_redirects.txt automatically changed after running yarn content move command

**Comment by teoli2003** [comment]
Yes, but you use the wrong capitalization, so redirect.txt has the wrong capitalization.

We used GitHub to propose a change, but we can't commit it via the GitHub interface.

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/_redirects.txt:5763]: Correct, but in https://github.com/mdn/content/pull/19660/commits/e847ec5cec82a507bb3bf1ce08099ef2b5401190 you hand-updated the slug ^^ Every time you change the slug, you have to update the redirects as well

**Comment by Josh-Cena** [comment]
You didn't do a clean merge.

File: files/en-us/_redirects.txt (hunk 3 above)
Comment (teoli2003) [inline files/en-us/_redirects.txt:12277]: This one needs to be updated too.

**Comment by iSatVeerSingh** [comment]
> You didn't do a clean merge.

This was a mistake because I don't know git commands very well.

**Comment by Josh-Cena** [comment]
Well—I see people having years of experience with git doing that as well 😄

**Review by teoli2003 — APPROVED** [review]
All look fine! (I've borked so many PRs with redirects and merges…)

**Comment by teoli2003** [comment]
Thank you!

**Pull request merged**